bootstrap.sh

```diff
@@ -93,6 +93,11 @@ then
   LIBTOOL_BIN="glibtool"
 else
   LIBTOOL_BIN="libtool"
+  lsb_release_id="$(lsb_release -si)"
+  if [ -n "${lsb_release_id}" ] \
```

Comment:
If this `-n` expression works (i.e. does not lead to shell errors when the lsb_release_id variable has an empty value, then the following `Debian` equality test will work as well. And if this `-n` expression does not work, then the following equality test will not work as well. Thus, you do not need the `-n` part of the test AFAICT.